validator/lib/readTileset.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
```

Comment:
Maybe just check that json.asset is defined since every tileset has an "asset" section. This should be good enough.